ext/dapr-ext-grpc/setup.cfg

```diff
@@ -7,7 +7,7 @@ license_file = LICENSE
 classifiers =
     Development Status :: 5 - Production/Stable
     Intended Audience :: Developers
-    License :: OSI Approved :: Apache License
+    License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
     Programming Language :: Python
     Programming Language :: Python :: 3.7
```

Comment:
While we're here, we should probably add 3.9 and 3.10

Reply:
Done